pubspec.yaml

```diff
@@ -38,6 +38,7 @@ dev_dependencies:
   mockito: ^5.4.4
   path: ^1.9.0
   zds_analysis: ^1.0.0
+  analyzer: ^6.7.0
```

Comment:
the action is failing because i have set it to keep these in alphabetical order - just move it up to the top to resolve this